option.go

```diff
@@ -210,13 +210,25 @@ func OptionSummary(summary string) func(*BaseRoute) {
 	}
 }
 
-// Description adds a description to the route.
+// Description sets the description to the route.
+// By default, the description is set by Fuego with some info,
+// like the controller function name and the package name.
+// If you want to add a description, please use [AddDescription] instead.
 func OptionDescription(description string) func(*BaseRoute) {
 	return func(r *BaseRoute) {
 		r.Operation.Description = description
 	}
 }
 
+// Description appends a description to the route.
+// By default, the description is set by Fuego with some info,
+// like the controller function name and the package name.
+func OptionAddDescription(description string) func(*BaseRoute) {
+	return func(r *BaseRoute) {
+		r.Operation.Description += description
```

Comment:
hmmm this is where a string builder could be useful. Could be annoying to need to remember the space or newline. But I guess the user would notice on the spec gen anyway 🤔